frontend/coprs_frontend/run/check_for_anitya_version_updates.py

```diff
@@ -80,12 +80,14 @@ def get_updated_packages(updates_messages, backend):
     for message in updates_messages:
         update = message['msg']
         project = update['project']
+        projectname = project['name'].lower()
```

Comment:
Not sure if we want to convert package names to lowercase but AFAIK PyPI packages are not case-sensitive so converting them or not converting them should both cause no problems.

I have no preference, I only noticed it

Reply:
Yea, mee neither - this was decided before - I now just moved the position of this code.